data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,114 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
```

Comment:
Not sure this ship should be in this file instead of one of the mission files.

Reply:
I dont think we put ship definitions in the files, only variants?
Also at some point we will probably have less powerful variants to sell I think